Various Gradle issues releasing 1.9

I add to disable various Gradle tests to release 1.9, namely:

 * some tests in the CLI module: https://github.com/quarkusio/quarkus/commit/b4551ff9f2ccd842b1f30b3b172514b3de6ae8c1
 * some tests in the Codestarts module: https://github.com/quarkusio/quarkus/commit/630a9d358ccfc1c5cf1771cad512df9960e0d18d
 * some tests in the Gradle IT: https://github.com/quarkusio/quarkus/commit/d9e4b44f747bf19bdf099e4d70d323402be3642b

There's a good chance it's due to similar causes i.e. deploying to a custom/empty repo (and maybe not having the artifacts in the default repo).

/cc @glefloch @aloubyansky

I have been able to fix tests disabled in :

- some tests in the CLI module: b4551ff
- some tests in the Codestarts module: 630a9d3


I still have a weird error in the GradleIT tests where it cannot find a dependency in my local maven repo: 
````
Caused by: org.gradle.internal.resolve.ArtifactNotFoundException: Could not find jboss-servlet-api_4.0_spec-1.0.0.Final.jar (org.jboss.spec.javax.servlet:jboss-servlet-api_4.0_spec:1.0.0.Final).
Searched in the following locations:
    file:/Users/glefloch/.m2.tmp/repo/org/jboss/spec/javax/servlet/jboss-servlet-api_4.0_spec/1.0.0.Final/jboss-servlet-api_4.0_spec-1.0.0.Final.jar
````

--- a/.github/workflows/release-build.yml
+++ b/.github/workflows/release-build.yml
@@ -1,0 +1,30 @@
+name: Quarkus Release Test Build
+
+on:
+  schedule:
+    - cron: '0 3 * * 2'
+env:
+  LANG: en_US.UTF-8
+jobs:
+  build:
+    name: "Prepare release"
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          ref: master
+      - name: Reclaim Disk Space
+        run: .github/ci-prerequisites.sh
+      - name: Set up JDK 8
+        # Uses sha for added security since tags can be updated
+        uses: joschi/setup-jdk@68381f2c0646f942f70b69f8e81fe10e1ed5d293
+        with:
+          java-version: 8
+      - name: Create maven repo
+        run: mkdir -p $HOME/release/repository
+      - name: Build and Test
+        run: |
+          mvn --settings .github/mvn-settings.xml \
+            -DskipITs -Dinvoker.skip=true \
+            -Dmaven.repo.local=$HOME/release/repository \
+            clean install

--- a/devtools/gradle/pom.xml
+++ b/devtools/gradle/pom.xml
@@ -85,7 +85,7 @@
                                 <argument>--no-daemon</argument>
                             </arguments>
                             <environmentVariables>
-                                <MAVEN_LOCAL_REPO>${settings.localRepository}</MAVEN_LOCAL_REPO>
+                                <MAVEN_REPO_LOCAL>${settings.localRepository}</MAVEN_REPO_LOCAL>
                                 <GRADLE_OPTS>${env.MAVEN_OPTS}</GRADLE_OPTS>
                             </environmentVariables>
                             <skip>${skip.gradle.build}</skip>

--- a/integration-tests/gradle/pom.xml
+++ b/integration-tests/gradle/pom.xml
@@ -61,7 +61,7 @@
                                         <argument>--stacktrace</argument>
                                     </arguments>
                                     <environmentVariables>
-                                        <MAVEN_LOCAL_REPO>${settings.localRepository}</MAVEN_LOCAL_REPO>
+                                        <MAVEN_REPO_LOCAL>${settings.localRepository}</MAVEN_REPO_LOCAL>
                                         <GRADLE_OPTS>${env.MAVEN_OPTS}</GRADLE_OPTS>
                                     </environmentVariables>
                                     <skip>${skip.gradle.tests}</skip>

